linux_os/guide/services/ssh/ssh_client/ssh_client_rekey_limit/oval/shared.xml

```diff
@@ -4,11 +4,13 @@
     <metadata>
       <title>{{{ rule_title }}}</title>
       {{{- oval_affected(products) }}}
-      <description>Ensure 'RekeyLimit' is configured with the correct value in /etc/ssh/ssh_config and /etc/ssh/ssh_config.d/*.conf</description>
+      <description>Ensure 'RekeyLimit' is configured with the correct value in /etc/ssh/ssh_config or /etc/ssh/ssh_config.d/*.conf</description>
     </metadata>
-    <criteria comment="RekeyLimit is correctly configured for ssh client" operator="AND">
-      <criterion comment="check that RekeyLimit is not configured in /etc/ssh/ssh_config" test_ref="test_ssh_client_rekey_limit_main_config" negate="true" />
-      <criterion comment="check correct RekeyLimit configuration in /etc/ssh/ssh_config.d/*.conf" test_ref="test_ssh_client_rekey_limit_include_configs" />
+    <criteria comment="RekeyLimit is correctly configured for ssh client" operator="OR">
```

Comment:
```suggestion
    <criteria comment="RekeyLimit is correctly configured for ssh client in the main configuration file" operator="OR">
```

Reply:
Actually, the first comment is correct as RekeyLimit is configured for sshd client OR in an includes file.

Reply:
ah you are right.